apps/movies/src/movies.repository.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, Logger } from "@nestjs/common";
+import { AbstractRepository } from "@app/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { EntityManager, Repository } from "typeorm";
+import { Movie } from "./entities/movie.entity";
+
```

Comment:
let's remove extra line 
actually prettier should do it, check prettier config

Reply:
https://github.com/simeontg/cinema-backend/pull/2/commits/8f8afa44598a10daf4bc0b631bf70e11c364603a